rpcs3/Emu/RSX/VK/VKGSRender.h

```diff
@@ -384,6 +384,7 @@ class VKGSRender : public GSRender, public ::rsx::reports::ZCULL_control
 
 	std::unique_ptr<vk::text_writer> m_text_writer;
 	std::unique_ptr<vk::upscaler> m_upscaler;
+	bool m_use_fsr_upscaling{false};
```

Comment:
There is no need to permanently store this

Reply:
I guess we need this to see the change in settings.
Is it possible to have a callback system for config settings? It would make things much cleaner for dynamic RSX options.
Like, if a checkbox value is changed, call some function.

Reply:
There's a function that is called when we save the settings dialog.
Idk if it's better to add that in

Reply:
Leave it as-is for now, that needs to be handled as a bigger job involving handling renderer restart gracefully.